wiki/Tournaments/CMT_4K/2017/en.md

```diff
@@ -44,29 +44,49 @@ Twitch: [aiyulu](https://www.twitch.tv/aiyulu).
 
 ## Match schedule
 
-### Round of 16
+### Quarterfinals
 
-Saturday, 8. July 2017  
+Saturday, 15. July 2017  
 
 | Match Time (UTC+8) | Team A | | | Team B | Match ID |
 | :--: | ---: | ---: | :--- | :--- | :--: |
-| 15:00 | drunkenstein | ![][HK] | ![][CN] | - ZedXD - | A |
-| 16:00 | [Crz]Riri | ![][CN] | ![][CN] | spensll | B |
-| 20:00 | NED_Q | ![][CN] | ![][CN] | wjh123 | C |
-| 21:00 | Stink God | ![][CN] | ![][CN] | [Crz]MemoryI | D |
+| 15:00 | drunkenstein | ![][HK] | ![][CN] | [Crz]Riri | I |
+| 16:00 | - ZedXD - | ![][CN] | ![][CN] | spensll | R |
+| 20:00 | wjh123 | ![][CN] | ![][CN] | Stink God | S |
+| 21:00 | NED_Q | ![][CN] | ![][CN] | [Crz]MemoryI | J |
 
-Sunday, 9. July 2017  
+Sunday, 16. July 2017  
 
 | Match Time (UTC+8) | Team A | | | Team B | Match ID |
 | :--: | ---: | ---: | :--- | :--- | :--: |
-| 13:00 | [Crz]Liusecy | ![][CN] | ![][CA] | Murasame | E |
-| 14:00 | Neokje | ![][MY] | ![][AU] | [Crz]Yukikaze- | F |
-| 15:00 | - Choma - | ![][TW] | ![][CN] | pikechu | G |
-| 20:00 | [Crz]ScSolAr | ![][CN] | ![][CN] | [Crz]Mix0130 | H |
-
+| 13:00 | Murasame | ![][CA] | ![][AU] | [Crz]Yukikaze- | T |
+| 15:00 | pikechu | ![][CN] | ![][CN] | [Crz]Mix0130 | U |
+| 20:00 | [Crz]Liusecy | ![][CN] | ![][MY] | Neokje | K |
+| 21:00 | - Choma - | ![][TW] | ![][CN] | [Crz]ScSolAr | L |
 
 ## Mappools
 
+### Quarterfinals
+
+[**Download all** (OneDrive)](https://1drv.ms/u/s!AuGED5b9f0uloUjg8WuPP1a5m3Gx)  
+- [DJ Genki feat. yukacco - Candy Star \[Noodle Star\]](https://osu.ppy.sh/b/1243769)
+- [Nauts - Second Run (Core Mix)  \[4K LN\]](https://osu.ppy.sh/b/1258021)
+- [Camellia - Singularity  \[Artificial Mind\]](https://osu.ppy.sh/b/1073952)
+- [Helblinde - The Solace of Oblivion \[A Lie's Asgard\]](https://osu.ppy.sh/b/1274057)
+- [Musical Masterpiece - MANIERA \[ilikexd's FOOLISH\]](https://osu.ppy.sh/b/535893)
+- [Nekomata L.E.D. Master+ - Chrono Diver -PENDULUMs- \[Rage!!\]](https://osu.ppy.sh/b/835621)
+- [Ryu* Vs. Sota - Go Ahead!! \[Legendaria\]](https://osu.ppy.sh/b/1212926)
+- [xi - Aragami \[Ultimate\]](https://osu.ppy.sh/b/757103)
+- [Omnipotenx - PEACE BREAKER \[NEGATIVE LEVEL 6\]](https://osu.ppy.sh/b/1215329)
+- [Demetori - Wind God Girl \[14\]](https://osu.ppy.sh/b/1306105)
+- [Gram - Sigmund \[Karasu's Leggendaria\]](https://osu.ppy.sh/b/1347575)
+- [Iced Blade feat. Rita - Sora no Kodou (short ver.) \[4K Liberation\]](https://osu.ppy.sh/b/942355)
+- [EYKHA & LOU & MERYL - Delta Decision \[Domination\]](https://osu.ppy.sh/b/1276313)
+- [t+pazolite - cheatreal \[TUTORIAL\]](https://osu.ppy.sh/b/873247)
+
+TB:
+- [Billain - Specialist \[Maestro\]](https://osu.ppy.sh/b/1216877)
+
 ### Round of 16
```

Comment:
The first instance of this was replaced to Quarterfinals. Should be done here too.